src/Sylius/Bundle/CoreBundle/Resources/config/services/providers.xml

```diff
@@ -67,5 +67,11 @@
             <argument type="service" id="sylius.repository.order" />
             <tag name="sylius.statistics.orders_totals_provider" type="year" />
         </service>
+
+        <service id="sylius.provider.configurable_grid_provider" class="Sylius\Bundle\CoreBundle\Grid\Provider\ConfigurableProvider">
+            <argument type="tagged_locator" tag="sylius.grid_provider" index-by="key" />
+            <argument>%sylius_core.grids_configuration%</argument>
+            <tag name="sylius.grid_provider" key="array" priority="100"/>
```

Comment:
Do you need to tag other exists (array/service) providers?

Reply:
Maybe, I don't know how this is integrated in the resource provider.